test/js/specs/happy.js

```diff
@@ -12,34 +12,120 @@ describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
   //Welcome Screen Tests
-  const copy = welcome.copy()
+  const copyWelcome = welcome.copyWelcome()
 
   it('test website title', async () => {
-    await driver.get(localhostUrl)
+    driver.get(localhostUrl)
     const title = driver.getTitle();
     expect(title).to.equal('Onfido SDK Demo');
   })
 
   it('test welcome screen title', async () => {
     const welcomeTitleText = welcome.welcomeTitle.getText()
-    expect(welcomeTitleText).to.equal(copy["title"]);
+    expect(welcomeTitleText).to.equal(copyWelcome["title"]);
     welcome.welcomeTitle.isDisplayed()
   })
 
   it('test welcome screen subtitle', async () => {
     const welcomeSubtitleText = welcome.welcomeSubtitle.getText()
-    expect(welcomeSubtitleText).to.equal(copy["description_p_1"] + "\n" + copy["description_p_2"]);
+    expect(welcomeSubtitleText).to.equal(copyWelcome["description_p_1"] + "\n" + copyWelcome["description_p_2"]);
     welcome.welcomeSubtitle.isDisplayed()
   })
 
   it('test verify identity button', async () => {
     const verifyIdentityBtnText = welcome.primaryBtn.getText()
-    expect(verifyIdentityBtnText).to.equal(copy["next_button"]);
+    expect(verifyIdentityBtnText).to.equal(copyWelcome["next_button"]);
     welcome.primaryBtn.isDisplayed()
   })
 
   it('test footer is displayed', async () => {
     welcome.footer.isDisplayed()
+   })
+
+  //Document selection screen
+  const copyDocumentSelection = documentSelection.copyDocumentSelection()
+  const copyDocuments = documentSelection.copyDocuments()
+
+  it('test document selection title', async () => {
```

Comment:
To me this can be tested within one test scenario, or using a context - now because nothing changes but with each test you're requiring a driver to load the page, you'd just making tests slower.

Reply:
This really depends on the approach we want to take.

I normally tend to keep each test simple and not to be responsible for multiple aspects.

Reply:
Yes, I agree. I've never worked with this test stack but I would say if there was a way of initialising the driver only once for each scenarios that would be ideal.

Reply:
Cool :) 
Addressed in 22419e6.